.env.example

```diff
@@ -11,3 +11,10 @@ REACT_APP_SENTRY_DSN=
 
 # Wallet Connect project ID
 REACT_APP_PROJECT_ID=e7e6e90622ec8e65c4d02e829a68529a
+
+# The RPC URL will be visible in the browser network tab (leaking the API key). It is advised to whitelist only API3 DAO
```

Comment:
Specifically [these addreses](https://github.com/api3dao/api3-dao-dashboard/blob/551c112e553fd9243274a13247d60a62475f7178/src/contract-deployments/mainnet-dao.json#L5)

Reply:
Can we make this a bit more explicit in the comment? Maybe a link to the contract addresses or just put them here since they won't change? Right now it's not obvious where to find them

Reply:
Did this 
![Screenshot 2023-11-06 at 10-39-35 Alchemy Dashboard](https://github.com/api3dao/api3-dao-dashboard/assets/19530665/52ded924-7fca-4d25-9d6f-c5d44184e1ed)